src/flavors/__tests__/fixtures.ts

```diff
@@ -46,6 +47,11 @@ export function createFixtures(
         @IsBoolean()
         requiredBooleanValue!: boolean;
 
+        @IsDate({
+            format: DateFormat.DATE,
```

Comment:
I have a few thoughts here:
 - We might want separate decorators for `IsDate` and `IsDateTime`.
 - We might want separate decorators for `Date` and date strings.
 - If we choose to keep the format, we might want to have a test that allows passing a string input; I'm not convinced that everyone will want to import and use the enum.